cpp/build.py

```diff
@@ -1,24 +1,115 @@
 #!/usr/bin/env python3
+"""
+CMake-based build script for lammps.js
+
+This script builds LAMMPS as a WebAssembly module using CMake and Emscripten.
+It handles cloning LAMMPS, configuring CMake, building the library, and linking
+the final WASM module.
+
+Usage:
+    python cpp/build.py           # Basic build
+    python cpp/build.py --debug   # Debug build with source maps
+    python cpp/build.py -r        # Force full recompilation
+
+Environment variables:
+    EMSDK_PATH    - Path to Emscripten SDK (required)
+    LAMMPS_TAG    - Git tag/branch for LAMMPS (default: patch_10Sep2025)
+    PACKAGES      - Space-separated list of LAMMPS packages (default: MOLECULE)
+    SINGLE_FILE   - Set to "0" to output separate .wasm file (default: "1")
+"""
+
 import os
-import shutil
 import subprocess
+import shutil
+import sys
 from pathlib import Path
 
+# Configuration
 LAMMPS_TAG = os.environ.get("LAMMPS_TAG", "patch_10Sep2025")
+PACKAGES = os.environ.get("PACKAGES", "MOLECULE").split()
```

Comment:
I think we should consider maybe deploying multiple versions with different set of packages. Not sure exactly how we should do it though.